code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -64,10 +65,7 @@
 	var/datum/personal_crafting/handcrafting
 
 /mob/living/simple_animal/drone/New()
-	..()
-	if(name != initial(name))
-		name = name + " ([rand(100,999)])"
-	real_name = name
+	. = ..()
```

Comment:
No, really, you can remove these two lines you don't *need* the new if this is all it's doing